### PR TITLE
#3555 add meta_example for classifier_perceptron undocumented-sample

### PR DESCRIPTION
**Pull request by grg121**
Hi! My name is Paco and I'm interested in this project and I would like to get involved with it.

I've created the sample and tested c and python result. They are both the same. 
I'm sending another PR to shogun-data with the integration testing data output of the c++ executable and I'll make a cookbook page then.

### CODE REVIEW

**Comment by karlnapf** [comment]
Welcome Paco, thanks for the patch! :)

**Comment by karlnapf** [comment]
The example looks good. I have in the meantime merged you `data` PR, so you can now add the update to this PR as well

**Comment by grg121** [comment]
Done! :) 

Thank you! Now I'm gonna write a Cookbook page.

**Comment by karlnapf** [comment]
Thanks for the contribution! :)

**Comment by karlnapf** [comment]
btw is there still an undocumented python example that can be deleted?

File: examples/meta/src/neural_nets/classification_perceptron.sg [hunk 1]
```diff
@@ -0,0 +1,35 @@
+CSVFile f_feats_train("../../data/classifier_binary_2d_linear_features_train.dat")
```
Comment (karlnapf) [inline examples/meta/src/neural_nets/classification_perceptron.sg]: ah, I had overseen this.
Can you pls send another PR where you  move the perceptron to the "binary" folder, it does not really follow the NN structure in Shogun. Same for the data file.
Also could you remove the python example in the same PR?

Comment (grg121) [reply examples/meta/src/neural_nets/classification_perceptron.sg]: Ahh ok, I thought it had to be with NN examples, I'll move it and remove the python example...

Also... I'm seeing that one check didn't pass which is weird because I just send the data file... don't know :/ 

Well, I'lll do that and send another commit!

Comment (grg121) [reply examples/meta/src/neural_nets/classification_perceptron.sg]: another PR* xp
